tests/integration_tests/test_collection_v1_compatibility.py

```
@@ -0,0 +1,497 @@
+
+        assert len(query_results["ids"][0]) >= 0  # May or may not find results
+
+        # Cleanup
+        db_client.delete_collection(collection_name)
+
+    def test_v1_collection_table_name_format(self, db_client):
+        """Test that v1 collections use correct table name format"""
+        collection_name = f"test_v1_table_{int(time.time() * 1000)}"
+
+        # Create v1 collection
+        collection = db_client.create_collection(
```

Ruff (0.14.13)

Local variable `collection` is assigned to but never used

Remove assignment to unused variable `collection`

(F841)